commit 48b64da89b3ecb7ccb576a334b19b3066dd2b793
Author: Takuto Ikuta <tikuta@chromium.org>
Date:   Wed Apr 30 02:36:55 2025 -0700

    chrome: Adds a missing #include for std::optional.
    
    This is to fix clang module build.
    
    Bug: 40263312
    Change-Id: I91953cc95459304a6a7b2073f9c5455d166883a4
    Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/6499673
    Commit-Queue: Colin Blundell <blundell@chromium.org>
    Commit-Queue: Takuto Ikuta <tikuta@chromium.org>
    Reviewed-by: Colin Blundell <blundell@chromium.org>
    Auto-Submit: Takuto Ikuta <tikuta@chromium.org>
    Cr-Commit-Position: refs/heads/main@{#1453791}

diff --git a/chrome/browser/ui/safety_hub/disruptive_notification_permissions_manager.h b/chrome/browser/ui/safety_hub/disruptive_notification_permissions_manager.h
index 00fe24ccb73ea..0bef7cbd7a1b1 100644
--- a/chrome/browser/ui/safety_hub/disruptive_notification_permissions_manager.h
+++ b/chrome/browser/ui/safety_hub/disruptive_notification_permissions_manager.h
@@ -6,6 +6,7 @@
 #define CHROME_BROWSER_UI_SAFETY_HUB_DISRUPTIVE_NOTIFICATION_PERMISSIONS_MANAGER_H_
 
 #include <memory>
+#include <set>
 
 #include "base/scoped_observation.h"
 #include "base/time/clock.h"
